src/api/Authentication/DatabaseAuthenticationClaimsTransformation.cs

```diff
@@ -56,6 +56,15 @@ public async Task<ClaimsPrincipal> TransformAsync(ClaimsPrincipal principal)
         user.FirstName = principal.Claims.FirstOrDefault(c => c.Type == ClaimTypes.GivenName)?.Value ?? user.FirstName;
         user.LastName = principal.Claims.FirstOrDefault(c => c.Type == ClaimTypes.Surname)?.Value ?? user.LastName;
         user.Name = $"{user.FirstName[0]}. {user.LastName}";
+
+        var emailClaim = principal.Claims.FirstOrDefault(c => c.Type == ClaimTypes.Email);
+        var emailVerifiedClaim = principal.Claims.FirstOrDefault(c => c.Type == "email_verified");
+
+        if (emailClaim == null || emailVerifiedClaim == null || emailVerifiedClaim.Value != "true")
```

Comment:
REQ;
Der Claim `email_verified` ist auf dem [Swisstopo AWS Cognito IDP](https://eu-west-1.console.aws.amazon.com/cognito/v2/idp/user-pools?region=eu-west-1) auf allen Umgebungen (REF, ABN, PROD) bei allen Benutzer die von externen Identities/Provider stammen (also alle SAML 2 eIAM Benutzer) auf `false`. Somit muss diese Validierung leider für den Moment raus.

@danjov Wir/Ihr müsst hier entscheiden ob für Benutzer aus dem eIAM der _E-Mail verified_-Schalter im Cognito auf `true` gesetzt werden kann/soll. Wie das umgesetzt werden kann, weiss ich grad nicht. Aus Security Sicht wäre es sicher wünschenswert, wenn man in der Applikation nur verifizierte E-Mails verwendet.

Reply:
@flenny @Lehats  Ist als https://github.com/swisstopo/swissgeol-boreholes-suite/issues/1838 erfasst und muss separat angeschaut werden. Bis dahin müssen wir die Validierung für den Moment weglassen.